src/main/resources/project/lib/EC/JBoss.pm

```diff
@@ -89,6 +92,12 @@ sub new {
     }
 
     my $creds = $self->get_credentials();
+
+    if ($creds->{log_level} && is_positive_int($creds->{log_level})) {
```

Comment:
Why are we reading the log_level from $creds? Are we getting the plugin configurations with the credentials call in $self->get_credentials()?